ui/appui-react/src/appui-react/widget-panels/ToolSettings.tsx

```diff
@@ -30,15 +30,25 @@ export interface ToolSettingsEntry {
   editorNode: React.ReactNode;
 }
 
-function EmptyToolSettingsLabel({ toolId }: { toolId: string }) {
+function EmptyToolSettingsLabel({
+  toolId,
+  customMessage,
+}: {
+  toolId: string;
+  customMessage?: string;
+}) {
   const { translate } = useTranslation();
   const tool = IModelApp.tools.find(toolId);
   const toolName = tool?.flyover;
```

Comment:
Instead of passing `customMessage` from the static property `InternalFrontstageManager.activeFrontstageDef` use
`useActiveFrontstageDef()` in `EmptyToolSettingsLabel`.